count/Feature/SessionSummaryFeature.swift

```diff
@@ -204,7 +222,7 @@ struct SessionSummaryView: View {
                 }
                 // TODO: Button: retry session with same settings (if timed session was completed)
```

Comment:
I considered adding a "Try Quiz Again" button, but I think it's better to add in a separate PR.